packages/compass/src/main/menu.ts

```diff
@@ -485,6 +505,31 @@ class CompassMenu {
       this.load(bw);
     });
 
+    app.on('auto-updater:new-state', (state) => {
+      const updateManagerState = ((): UpdateManagerState => {
+        switch (state) {
+          case AutoUpdateManagerState.Initial:
+          case AutoUpdateManagerState.Disabled:
+          case AutoUpdateManagerState.UserPromptedManualCheck:
+          case AutoUpdateManagerState.CheckingForUpdatesForManualCheck:
+          case AutoUpdateManagerState.CheckingForUpdatesForAutomaticCheck:
+          case AutoUpdateManagerState.NoUpdateAvailable:
+          case AutoUpdateManagerState.UpdateAvailable:
+          case AutoUpdateManagerState.UpdateDismissed:
+          case AutoUpdateManagerState.DownloadingError:
+          case AutoUpdateManagerState.Restarting:
+            return 'idle';
+          case AutoUpdateManagerState.ManualDownload:
+          case AutoUpdateManagerState.DownloadingUpdate:
+            return 'installing updates';
+          case AutoUpdateManagerState.RestartDismissed:
+          case AutoUpdateManagerState.PromptForRestart:
+            return 'ready to restart';
+        }
+      })();
```

Comment:
Either way, do you have a preference?

I chose this approach because it felt strange for the auto updater to be defining two different sets of states(the actual states and the "coarser" states), the coarser only being used inside of the menu.

Reply:
Well, yeah, mild preference for moving this to auto-update-manager.ts, but I'm also fine either way, really.

I wouldn't really consider this adding a new set of states; more like a utility to "categorize"/group the actual auto updater states.